### ISSUE TITLE
Error after log in

### ISSUE THREAD

**Issue by privatgt** [nextcloud/android#10120]
### Cause of error
```java
Exception in thread "main" java.lang.RuntimeException: Unable to start activity ComponentInfo{com.nextcloud.client/com.owncloud.android.ui.activity.FileDisplayActivity}: java.util.NoSuchElementException: No value present
    at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3270)
    at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3409)
    at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:83)
    at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
    at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
    at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2016)
    at android.os.Handler.dispatchMessage(Handler.java:107)
    at android.os.Looper.loop(Looper.java:214)
    at android.app.ActivityThread.main(ActivityThread.java:7356)
    at java.lang.reflect.Method.invoke(Native Method)
    at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:491)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:940)
Caused by: Exception in thread "main" java.util.NoSuchElementException: No value present
    at com.nextcloud.java.util.Optional.get(Optional.java:142)
    at com.nextcloud.client.mixins.SessionMixin.setAccount(SessionMixin.kt:71)
    at com.nextcloud.client.mixins.SessionMixin.onCreate(SessionMixin.kt:134)
    at com.nextcloud.client.mixins.MixinRegistry.onCreate(MixinRegistry.kt:56)
    at com.owncloud.android.ui.activity.BaseActivity.onCreate(BaseActivity.java:65)
    at com.owncloud.android.ui.activity.ToolbarActivity.onCreate(ToolbarActivity.java:78)
    at com.owncloud.android.ui.activity.DrawerActivity.onCreate(DrawerActivity.java:915)
    at com.owncloud.android.ui.activity.FileActivity.onCreate(FileActivity.java:195)
    at com.owncloud.android.ui.activity.FileDisplayActivity.onCreate(FileDisplayActivity.java:242)
    at android.app.Activity.performCreate(Activity.java:7824)
    at android.app.Activity.performCreate(Activity.java:7813)
    at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1306)
    at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3245)
    at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3409)
    at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:83)
    at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
    at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
    at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2016)
    at android.os.Handler.dispatchMessage(Handler.java:107)
    at android.os.Looper.loop(Looper.java:214)
    at android.app.ActivityThread.main(ActivityThread.java:7356)
    at java.lang.reflect.Method.invoke(Native Method)
    at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:491)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:940)

```

### App information
* ID: `com.nextcloud.client`
* Version: `30200052`
* Build flavor: `generic`

### Device information
* Brand: `Xiaomi`
* Device: `davinci`
* Model: `Mi 9T`
* Id: `QQ3A.200805.001`
* Product: `davinci`

### Firmware
* SDK: `29`
* Release: `10`
* Incremental: `eng.root.20220415.091244`

**Comment by privatgt** [comment]
In previous version working fine
I downgraded

**Comment by joshtrichards** [comment]
Fixed by #9710
